Lib/test/test_logging.py

```diff
@@ -5049,21 +5049,21 @@ def outer():
 
         records = self.recording.records
         outer()
-        self.assertEqual(records[-1].funcName, 'innermost')
+        self.assertEqual(records[-1].funcName, 'LoggerTest.test_find_caller_with_stacklevel.<locals>.innermost')
```

Comment:
This makes the line 112 chars.  Suggest either
```suggestion
        context = 'LoggerTest.test_find_caller_with_stacklevel.<locals>.'
        self.assertEqual(records[-1].funcName, context + 'innermost')
```
or
```suggestion
        self.assertEqual(
            records[-1].funcName,
             'LoggerTest.test_find_caller_with_stacklevel.<locals>.innermost')
```
and repeat below

Reply:
Logging is no longer affected after I restricted the scope of this PR to just tracing. I will address this in a follow up.